osu.Server.Spectator/Database/OsuDatabase.cs

```diff
@@ -4,9 +4,9 @@
 using System;
 using MySqlConnector;
 
-namespace osu.Server.Spectator
+namespace osu.Server.Spectator.Database
 {
-    public class Database
+    public class OsuDatabase
```

Comment:
`DatabaseHelper` maybe? or just `DB` (similar to `RNG`) since this is intended to be a concise call.

Reply:
I've gone with `DB`. I have a strong dislike for class names with `Helper` in them, always feels like open season for a random grab-bag of unrelated stuff.